src/main/java/store/controller/CategoryController.java

```diff
@@ -0,0 +1,19 @@
+package store.controller;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class CategoryController {
+
+    @Value("#{'${categories.list}'.split(',')}")
+    List<String> categories;
```

Comment:
Test doesnt have an access to spring props.
Search in google how to set value to private field